lib/rust/json-rpc/Cargo.toml

```diff
@@ -11,6 +11,8 @@ crate-type = ["cdylib", "rlib"]
 enso-prelude = { path = "../prelude", features = ["futures"]}
 enso-shapely = { path = "../shapely"}
 enso-web = { path = "../web" }
+enso-profiler-data = {path = "../profiler/data"}
```

Comment:
What depends on this? This crate is only intended to be used when reading profile files and analyzing them, it shouldn't be needed by anything at runtime

Reply:
This is actually not needed .